software/panora.yml

```diff
@@ -0,0 +1,12 @@
+name: "Panora"
+website_url: "https://panora.dev"
+source_code_url: "https://github.com/panoratech/Panora"
+description: "An API to add an integration catalog to your SaaS product in minutes. OSS Alternative to Merge.dev"
```

Comment:
```suggestion
description: "An API to add an integration catalog to your SaaS product in minutes. (alternative to Merge.dev)"
```

Reply:
Just cleaning up the description at the end, everything else looks good from my end!

Reply:
thank you @kokomo123!